src/includes/performance/traces-sample-rate/android.mdx

```diff
@@ -0,0 +1,26 @@
+```xml {filename:AndroidManifest.xml}
+<application>
+  <meta-data android:name="io.sentry.traces.sample-rate" android:value="0.2" />
+</application>
+```
+
+Or, if you are manually instrumenting Sentry:
+
+```java {tabTitle:Java}
+import io.sentry.android.core.SentryAndroid;
+
+SentryAndroid.init(this,
+  options -> {
+    ...
```

Comment:
```suggestion
```

Reply:
this is in all the performance Java/Kotlin examples, if we decide to remove it, I'd do a single PR.